features/cli/deploy.feature

```diff
@@ -46,7 +46,10 @@ Feature: deployment related features
     Then the output should contain "rolled out"
     # Wait the deployment till complete
     And the pod named "hooks-1-deploy" becomes ready
+    Given I wait for the steps to pass:
```

Comment:
@zhouying7780, Don't need to wrap it around a wait since the step `Given I wait for the pod named "hooks-1-deploy" to die` already has a built-in timeout.

Reply:
@pruan-rht thanks, have updated.